libr/include/r_core.h

```diff
@@ -287,6 +287,10 @@ R_API ut32 r_core_file_cur_fd (RCore *core);
 
 R_API void r_core_debug_rr (RCore *core, RReg *reg);
 
+/* project */
+R_API bool r_core_project_load(RCore *core, const char *prjfile);
+R_API bool r_core_xrefs_load(RCore *core, const char *prjfile);
```

Comment:
maybe that function should be named like `r_core_project_load_xrefs`. so we can keep all the tiny pieces of loading a project under the same naming prefix.